Frameworks/CoreGraphics/CGContextCairo.mm

```diff
@@ -105,6 +105,11 @@
 // An OSS request has been submitted. The link for the request is:
 // https://osstool.microsoft.com/palamida/RequestDetails.htm?rid=40072&projectId=1
 void CGContextCairo::_cairoImageSurfaceBlur(cairo_surface_t* surface) {
+    // TODO #1635: It's unsafe to edit the bitmap not through D2D1RenderTarget, during a BeginDraw()
+    // Escape any Begin/EndDraw() pairs for the lifetime of this function (to be removed during the CGD2D merge)
+    _CGContextEscapeBeginEndDrawStack(_rootContext);
```

Comment:
tried it, but it obscures too much for my taste.

---
In reply to: [96971228](https://github.com/Microsoft/WinObjC/pull/1705#discussion_r96971228) [](ancestors = 96971228)